swan-lake/development-tutorials/build-and-run/cicd.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: ballerina-cli-documentation-left-nav-pages-swanlake
+title: CI/CD for Ballerina Projects
```

Comment:
Can we change this to something like "Set Up CI/CD for Ballerina projects" or "Configure CI/CD for Ballerina Projects"?